cmd/acr/login.go

```diff
@@ -62,23 +66,32 @@ func newLoginCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "the registry username")
 	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "the registry password or identity token")
 	cmd.Flags().BoolVarP(&opts.fromStdin, "password-stdin", "", false, "read password or identity token from stdin")
+	cmd.Flags().BoolVarP(&opts.insecure, "insecure", "k", false, "allow connections to SSL registry without certs")
+	cmd.Flags().BoolVarP(&opts.plainHTTP, "plain-http", "", false, "allow insecure connections to registry without SSL")
+	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", false, "verbose output")
 	return cmd
 }
 
-func runLogin(opts loginOpts) error {
+func runLogin(opts loginOpts) (err error) {
+	var logLevel logrus.Level
 	if opts.debug {
-		logrus.SetLevel(logrus.DebugLevel)
+		logLevel = logrus.DebugLevel
+	} else if opts.verbose {
+		logLevel = logrus.InfoLevel
+	} else {
+		logLevel = logrus.WarnLevel
 	}
+	ctx, _ := orasgoAuth.WithLoggerLevel(context.Background(), logLevel)
```

Comment:
It seems not right as it changes the original behavior of other code.

Reply:
changed this part back to the original.